packages/glyphs-block/src/helpers.ts

```diff
@@ -0,0 +1,37 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { Choice } from '@frontify/guideline-blocks-settings';
+import { ApiFontStyle } from './types';
+
+export const getFonts = async (projectId: number): Promise<Choice[]> => {
+    const response = await fetch(`/api/project-font-family?project_id=${projectId}`);
+    const responseJson = await response.json();
+    const fonts = responseJson.data.map((fontStyle: ApiFontStyle) => ({
```

Comment:
add typing here to avoid any.

like:

```
const fonts: { label: string; value: string }[] = responseJson.data.map((fontStyle: ApiFontStyle) => ({
        label: fontStyle.name,
        value: fontStyle.css_family_name,
    }));

    return fonts.filter((font, index, self) => self.findIndex(({ value }) => value === font.value) === index);
    ```

Reply:
Optimized it 👍🏻